packages/components/docs/badges-addon.stories.tsx

```diff
@@ -0,0 +1,15 @@
+import { BADGE } from "@geometricpanda/storybook-addon-badges";
+import { StoryObj } from "@storybook/react";
+import React from "react";
+
+const Example = () => <p>Example to show badges addon</p>;
+
+export default {
```

Comment:
Is this mostly for internal documentation? Would suggest naming it `title: "Documentation/BadgesAddon"` (or once we merge with `next`, adding to one of the comprehensive docs files there)